WcaOnRails/config/locales/en.yml

```diff
@@ -1021,6 +1022,31 @@ en:
         content:
           #context: Yup, nothing to translate here! Just hit "Use the original version"
           '1': "%{current_user_claim}"
+      '9':
+        title: "I am registered for a competition and want to sign up for an additional event, what should I do? Can I bring guests?"
+        #context: Answer to "I want to sign up or an additional event ...?"
+        content:
+          '1': "Please contact the organizer. You can find the contact information on the competition website. If you are unsure where to find it, check the <a href='/competitions'>list of competitions</a>."
+      'regulations1':
+        title: "Is my puzzle competition legal?"
+        #context: Answer to "Is my puzzle competition legal?"
+        content:
+          '1': "You can find the regulations regarding puzzles in <a href='https://www.worldcubeassociation.org/regulations/#3'>Article 3</a>. All puzzles have to be approved by the WCA Delegate of the competition. If you are unsure, please ask him. To be on the safe side, you should bring a replacement puzzle in case you are not allowed to use it."
+      'regulations2':
+        title: "May I use a magnetic puzzle in an official WCA competition?"
+        #context: Answer to "May I use a magnetic puzzle in an official WCA competition?"
+        content:
+          '1': "You are allowed to use magnetic puzzles as long as they comply with the <a target='_blank' href='https://www.worldcubeassociation.org/regulations/'>WCA Regulations</a>. Please pay particular attention to <a href='https://www.worldcubeassociation.org/regulations/#3h'>Paragraph 3h</a>."
+      'regulations3':
+        title: "I lost a center cap or a piece of my puzzle. Am I still allowed to use it?"
+        #context: Answer to "Am I allowed to use damaged puzzles?"
+        content:
+          '1': "You are not allowed to use this puzzle. Puzzles must not have any damages or other differences that significantly distinguish any piece from a similar piece (see <a href='https://www.worldcubeassociation.org/regulations/#3j'>Regulation 3j</a>)."
```

Comment:
nit: Earlier you referred to "paragraph 3h", but here you refer to "regulation 3j". Are these regulations or paragraphs?